plugins/CoreVue/Commands/Build.php

```diff
@@ -123,13 +123,14 @@ private function watch($plugins, $printBuildCommand, OutputInterface $output)
             $output->writeln("<comment>$command</comment>");
             return;
         }
-        passthru($command);
+
+        passthru($commandSingle);
```

Comment:
Was that change intended? I didn't test it, but looks like it would try to execute the command containing placeholders now.

Reply:
no not intentional :+1: